docs/sources/dashboards/share-dashboards-panels/index.md

```diff
@@ -54,6 +54,9 @@ Grafana enables you to share dashboards and panels with other users within an or
 - A Snapshot
```

Comment:
nit: I saw below we are not capitalizing the first letter on snapshots. Should we lowercase it here?